vsintegration/deployment/WDExpressDeployTemplates/extension.vsixlangpack

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageLanguagePackManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011">
 	<Metadata>
-		<Identity Id="Microsoft.VisualStudio.WdExpressDeployment" Version="14.0" Language="neutral" Publisher="Microsoft" />
+		<Identity Id="Microsoft.VisualStudio.WdExpressDeployment" Version="15.0" Language="neutral" Publisher="Microsoft" />
```

Comment:
This will target Dev 15, and all templates will specifically look for 4.1 paths, which ship with Dev 15.